crates/builder/src/tasks/submit.rs

```diff
@@ -59,6 +66,26 @@ impl SubmitTask {
         serde_json::from_slice(&body).map_err(Into::into)
     }
 
+    async fn fetch_oauth_token(
+        &self,
+    ) -> eyre::Result<
+        StandardTokenResponse<oauth2::EmptyExtraTokenFields, oauth2::basic::BasicTokenType>,
```

Comment:
replace qualified path with use statement for the inner types

Reply:
i am unsure i know what this means